pleskdistup/common/src/files.py

```diff
@@ -46,9 +46,6 @@ def push_front_strings(filename: str, strings: typing.List[str]) -> None:
 
 
 def rewrite_json_file(filename: str, jobj: typing.Union[dict, typing.List]) -> None:
-    if filename is None or jobj is None:
```

Comment:
It might be better to raise an exception here

Reply:
> It might be better to raise an exception here

Arguments are statically annotated as not None and actually we don't check arguments dynamically for None anywhere in this file.